crates/matrix-sdk-crypto/src/store/crypto_store_wrapper.rs

```diff
@@ -126,6 +126,26 @@ impl CryptoStoreWrapper {
 
         self.store.save_changes(changes).await?;
 
+        // If we updated our own public identity, log it for debugging purposes
+        if tracing::level_enabled!(tracing::Level::DEBUG) {
+            for updated_identity in
+                identities.new.iter().chain(identities.changed.iter()).filter_map(|id| id.own())
```

Comment:
Took me sometime to see why this was just own identity. I sort of expected a have a filter on `user_id() ==` but I guess it's fine